core/pom.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
   <properties>
     <staplerFork>true</staplerFork>
-    <stapler.version>1.234</stapler.version>
+    <stapler.version>1.235-20150312.163208-1</stapler.version>
```

Comment:
Need to revert, and add a temporary explicit dep on `org.kohsuke.stapler:stapler:1.235-20150312.163158-1` in this POM.